config/routes.rb

```diff
@@ -29,27 +29,27 @@
   delete "route/:id", to: 'route#destroy', as: :route_delete
   get 'route/showEditRoute/:id', to: 'route#showEditRoute', as: :route_show_edit_route
   get "route/routeDetails/:id", to: 'route#routeDetails', as: :route_route_details
-  
-  post 'route/removeRouteBusstop/:id', to: 'route#removeRouteBusstop', as: :route_remove_route_busstop 
-  post 'route/includeRouteBusstop/:id', to: 'route#includeRouteBusstop', as: :route_include_route_busstop 
-  
+
+  post 'route/removeRouteBusstop', to: 'route#removeRouteBusstop', as: :route_remove_route_busstop
+  post 'route/includeRouteBusstop', to: 'route#includeRouteBusstop', as: :route_include_route_busstop
+
   post 'route/saveBusStopOnRoute', to: 'route#saveBusStopOnRoute'
   get 'welcome', to: 'welcome#welcome', as: :welcome_welcome
 
-  root 'front_ends#index'
+  root 'welcome#welcome'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.